security/certificate_types_descriptions/bootstrap-certificates.adoc

```diff
@@ -23,7 +23,7 @@ The kubelet-managed certificate is valid for one year and rotates automatically
 
 [NOTE]
 ====
-OpenShift Lifecycle Manager (OLM) does not update the certificates of Operators that it manages in proxy environments. These certificates must be managed by the user using the subscription config.
+OpenShift Lifecycle Manager (OLM) does not update the Bootstrap CA. The Bootstrap CA must be managed by the user using the subscription config.
```

Comment:
> The Bootstrap CA must be managed by the user using the subscription config.

What's the `the subscription config` means here? OLM managed subscription doesn't manage the `Bootstrap CA`.

Reply:
@jianzhangbjz I updated the PR after discussing it with the bug reporter. Can you PTAL?

Reply:
Sorry, I missed it. LGTM.